recipe/meta.yaml

```
@@ -30,6 +30,7 @@ test:
   requires:
     - pip
     - pytest
+    - matplotlib-base
   commands:
     - pip check
     - pytest -vv . --ignore=tests/test_diagram.py -k "not (TestExamples or Test02_WithoutPackrat or Test04_WithPackrat or Test06_WithBoundedPackrat or Test08_WithUnboundedPackrat or Test09_WithLeftRecursionParsing or Test10_WithLeftRecursionParsingBoundedMemo)"
```

are these tests still failing or are they fixed with the introduction of `matplotlib-base`?

---

Looks like all of those tests require a dependency we don't have ('railroad')

---

```suggestion
    # skip tests that require 'railroad', unavailable
    - pytest -vv . --ignore=tests/test_diagram.py -k "not (TestExamples or Test02_WithoutPackrat or Test04_WithPackrat or Test06_WithBoundedPackrat or Test08_WithUnboundedPackrat or Test09_WithLeftRecursionParsing or Test10_WithLeftRecursionParsingBoundedMemo)"
```

ah cool, thanks for checking, added a comment